package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "babel src  --out-dir production",
     "start": "node production/src/index.js",
+    "migrate": "babel-node ./src/database/tables.js",
```

Comment:
I can't find `src/databases/tables.js` file. Sequelize should handle the migration. Read more here https://sequelize.org/master/manual/migrations.html

Reply:
Okay let me read it